### DISCUSSION TITLE
Should we release 0.73.6?

### DISCUSSION THREAD

**Discussion by lunaleaps** [reactwg/react-native-releases#107] [Patches]
# All pick requests should now be filed through the [issue template](https://github.com/reactwg/react-native-releases/issues/new?assignees=&labels=Type%3A+Pick+Request&projects=&template=pick_request_form.yml)

You can see outstanding pick requests for 0.73.x in its [dedicated project board](https://github.com/orgs/reactwg/projects/6/views/3)

See https://github.com/reactwg/react-native-releases/discussions/117 for more details

We will lock this discussion for now.

---

Current Release: [0.73.5](https://github.com/facebook/react-native/releases/tag/v0.73.5)

Conversations on this thread are limited:

- [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking).
- [qualified cherry-pick requests](https://reactnative.dev/contributing/release-faq#what-is-a-qualified-pick-request) of commits on main that [did not make the previous patch version](https://reactnative.dev/contributing/release-faq#how-do-i-know-if-my-fixfeature-is-in-a-certain-release).

Please include a link to the specific commit on main to be cherry-picked, for example: [facebook/react-native@bd2b7d6](https://github.com/facebook/react-native/commit/20b0eba581a00e5e7e300f6377379b836617c147)

In other words, if you cannot point to a particular commit on main, then your request likely belongs as a new issue.
If the issue is a [major release issues](https://reactnative.dev/contributing/release-faq#what-is-release-blocking), please reference the issue here.

**Comment by cipolleschi** [comment]
Please, merge https://github.com/facebook/react-native/pull/43246
This solve a possible ENV var clash between a RN env var and other env vars users can set in their system.

**Reply by lunaleaps** [reply]
I've put it as a pick for 0.73.6 in the project board

**Comment by jankosecki** [comment]
Can we please pick https://github.com/facebook/react-native/commit/5fbdc994f945ee8d834632649e3b82451660e8fe which fixes https://github.com/facebook/react-native/issues/42604

**Reply by lunaleaps** [reply]
I've put it as a pick in the 0.73 project board https://github.com/orgs/reactwg/projects/6/views/3

**Comment by GijsWeterings** [comment]
(Copying from the top post, in case people scroll down immediately):

All pick requests should now be filed through the [issue template](https://github.com/reactwg/react-native-releases/issues/new?assignees=&labels=Type%3A+Pick+Request&projects=&template=pick_request_form.yml)

You can see outstanding pick requests for 0.73.x in its [dedicated project board](https://github.com/orgs/reactwg/projects/6/views/3)

See https://github.com/reactwg/react-native-releases/discussions/117 for more details

We will lock this discussion for now.
